plugins/internal/nexus-atlas-plugin/src/main/java/org/sonatype/nexus/atlas/internal/SystemInformationGeneratorImpl.groovy

```diff
@@ -47,22 +48,27 @@ class SystemInformationGeneratorImpl
 
   private final Provider<SystemStatus> systemStatusProvider
 
-  private final Map<String, String> parameters;
+  private final Map<String, String> parameters
 
-  private final List<PluginIdentity> pluginIdentities
+  private final BundleContext bundleContext
+
+  private final BundleService bundleService
 
   @Inject
   SystemInformationGeneratorImpl(final GlobalComponentLookupHelper componentLookupHelper,
                                  final ApplicationDirectories applicationDirectories,
                                  final Provider<SystemStatus> systemStatusProvider,
                                  final @Parameters Map<String, String> parameters,
-                                 final List<PluginIdentity> pluginIdentities)
+                                 final BundleContext bundleContext)
   {
     this.componentLookupHelper = checkNotNull(componentLookupHelper)
     this.applicationDirectories = checkNotNull(applicationDirectories)
     this.systemStatusProvider = checkNotNull(systemStatusProvider)
-    this.parameters = checkNotNull(parameters);
-    this.pluginIdentities = checkNotNull(pluginIdentities)
+    this.parameters = checkNotNull(parameters)
+    this.bundleContext = checkNotNull(bundleContext)
```

Comment:
it's also used later on in the `bundleContext.bundles.each` loop, though that could always be replaced by `bundleService.selectBundles(null, true).each`